docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.md

```diff
@@ -0,0 +1,422 @@
+- [kubectl (1.28+)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+- (optional) Domain name/[hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-working-with.html) in Route53. This allows you to expose Camunda 8 and connect via [zbctl](../../../../../../apis-tools/cli-client/) or [Camunda Modeler](https://camunda.com/download/modeler/).
+
+## Considerations
+
+While this guide is primarily tailored for UNIX systems, it can also be run under Windows by utilizing the [Windows Subsystem for Linux](https://learn.microsoft.com/windows/wsl/about).
+
+### Architecture
+
+Note the [existing Architecture](./../../../../platform-architecture/overview.md#architecture) extended by deploying a Network Load Balancer with TLS termination within the [ingress](https://kubernetes.github.io/ingress-nginx/user-guide/tls/) below.
+
+Additionally, two components ([external-dns](https://github.com/kubernetes-sigs/external-dns) and [cert-manager](https://cert-manager.io/)) handle requesting the TLS certificate from [Let's Encrypt](https://letsencrypt.org/) and configuring Route53 to confirm domain ownership and update the DNS records to expose the Camunda 8 deployment.
+
+![Camunda 8 Self-Managed AWS Architecture Diagram](./assets/camunda-8-self-managed-architecture-aws.png)
+
+## Usage
+
+In the following, we're using `helm upgrade --install` as it runs install on initial deployment and upgrades future usage. This may make it easier for future [Camunda 8 Helm upgrades](../../../../platform-deployment/helm-kubernetes/upgrade.md) or any other component upgrades.
+
+### Environment prerequisites
+
+To streamline the execution of the subsequent commands, it is recommended to export multiple environment variables.
+
+The following are the required environment variables with some example values:
+
+```shell
+# Your standard region that you host AWS resources in
+export REGION=eu-central-1
+# Following two environment variables can be skipped if you don't have a domain
+# The domain name that you intend to use
+export DOMAIN_NAME=camunda.example.com
+# The e-mail to register with Let's Encrypt
+export MAIL=admin@camunda.example.com
+# The Ingress-Nginx Helm Chart version
+export INGRESS_HELM_CHART_VERSION="4.8.3"
+# The External DNS Helm Chart version
+export EXTERNAL_DNS_HELM_CHART_VERSION="1.13.1"
+# The Cert-Manager Helm Chart version
+export CERT_MANAGER_HELM_CHART_VERSION="1.13.2"
+# The Camunda 8 Helm Chart version
+export CAMUNDA_HELM_CHART_VERSION="8.3.3"
+```
+
+Additionally, follow the guide from either [eksctl](./eks-helm.md) or from [Terraform](./terraform-setup.md) to retrieve the following values, which will be required for subsequent steps:
+
+- EXTERNAL_DNS_IRSA_ARN
+- CERT_MANAGER_IRSA_ARN
+- DB_HOST
+- PG_USERNAME
+- PG_PASSWORD
+- DEFAULT_DB_NAME
+- REGION
+
+### DNS set up
+
+:::info
+If you don't have a domain name, you can skip to [Camunda 8](#camunda-8).
```

Comment:
Can we describe what does it mean? If you don't have domain, you will not be able to access C8 web endpoints from the outside of the AWS VPC...